src/Service/DocumentUploader.php

```diff
@@ -38,6 +39,41 @@ public function __construct(SluggerInterface $slugger, string $uploadDocumentDir
         $this->mimeTypeGuesser = $mimeTypeGuesser;
     }
 
+    /**
+     * Creates and returns new document from filename.
+     */
+    public function createDocumentFromPath(string $fileName, User $user, string $documentName, string $documentType): Document
+    {
+        $document = new Document();
+
+        $document->setFilename($fileName);
+        $document->setDocumentName($documentName);
+        $document->setPath($this->getFilepathFromProjectDirectory($fileName));
+        $document->setUploadedBy($user);
+        $document->setType($documentType);
+
+        return $document;
+    }
+
+    /**
+     * Creates, uploads and returns new document from a file.
+     */
+    public function createDocumentFromFile(UploadedFile $file, User $user, string $documentName, string $documentType): Document
```

Comment:
I think this function should be called `createDocumentFromUploadedFile` to make it clear what is does (I was confused about the difference between `createDocumentFromPath` and `createDocumentFromFile` before I came here).

Reply:
Changed name as proposed, this does indeed make it much clearer.